src/components/new-members/index.js

```diff
@@ -9,8 +9,15 @@ import { userContext } from '@store/user/user-context';
 
 // returns card which shows details of new member
 const renderNewUserCard = (newMember) => {
+  const { isSuperUser } = userContext();
   return (
-    <div className={styles.containerForNewMember}>
+    <div
+      className={
+        isSuperUser
+          ? styles.containerForNewMemberHover
```

Comment:
Can you please think of better classnames here? Rest everything looks great!

Reply:
Sure, I am changing it to something more meanigful.